src/web/app/pages-instructor/instructor-student-list-page/instructor-student-list-page.component.html

```diff
@@ -1,157 +1,75 @@
-<h1> Student List</h1>
-
-<div class="card bg-light mb-4 pb-3">
-  <div class="card-body">
-    <div class="form-group">
-      <div class="input-group">
-        <input type="text" class="form-control" [(ngModel)]="searchQuery" (keyup.enter)="search()"
-               placeholder="Charlie Barnes">
-        <div class="input-group-append">
-          <button class="btn btn-primary" (click)="search()">Search</button>
-        </div>
+<h1>Student List</h1>
+<p>
+  This page gives the student details for all your courses. Expand the cards below to view your students. Use the search function if you need to filter for a specific student.
```

Comment:
Maybe this is better? "...Use the [search function](...) if you need to find a specific student.", and then put a hyperlink from search function to the `Search` page

Reply:
Yes good point! Updated